<launch>
  <!-- use sim time -->
  <param name="use_sim_time" value="true"/>

  <!-- Robot pose ekf -->
  <node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf" output="screen">
    <param name="freq" value="30.0"/>
    <param name="sensor_timeout" value="1.0"/>
    <param name="odom_used" value="true"/>
    <param name="imu_used" value="true"/>
    <param name="vo_used" value="false"/>
    <remap from="odom" to="base_odometry/odom" />
    <remap from="imu_data" to="torso_lift_imu/data" />
  </node>

  <node pkg="rosbag" name="rosbag" type="play" args="--clock --hz 100 -d .4 $(find robot_pose_ekf)/../../../build/navigation/robot_pose_ekf/test/ekf_test2.bag" />

  <test test-name="test_robot_pose_ekf" pkg="robot_pose_ekf" type="test_robot_pose_ekf" time-limit="120" />
</launch>
